optaplanner-core/src/test/java/org/optaplanner/core/api/solver/SolverManagerTest.java

```diff
@@ -157,6 +163,20 @@ public void exceptionInConsumer() throws InterruptedException {
         assertEquals(SolverStatus.NOT_SOLVING, solverJob1.getSolverStatus());
     }
 
+    @Test
+    public void solveGenerics() {
+        final SolverConfig solverConfig = PlannerTestUtils
+                .buildSolverConfig(TestdataSolution.class, TestdataEntity.class);
+        SolverManager<TestdataSolution, Long> solverManager = SolverManager
+                .create(solverConfig, new SolverManagerConfig());
+
+        BiConsumer<Object, Object> exceptionHandler = (o1, o2) -> fail("This is unexpected");
```

Comment:
@ge0ffrey This type of exception handler couldn't be used with the `solve()` method.